downloads/index.md

```diff
@@ -3,4 +3,138 @@ layout: custom
+                                <p>{% t downloads.blockchain1 %} <a href="https://downloads.getmonero.org/blockchain.raw">{% t downloads.blockchain2 %}</a> {% t downloads.blockchain3 %}</p>
+                            </div>
+                        </div>
+                    {% elsif data_downloads.id == "hardware" %}
+                        <div class="row">
+                            <div class="col-md-8 col-md-offset-2 col-sm-12 col-xs-12">
+                                <p>{% t downloads.hardware1 %} <a href="https://forum.getmonero.org/9/work-in-progress/88149/dedicated-monero-hardware-wallet" target="_blank" rel="noreferrer, noopener">{% t downloads.hardware2 %}</a> {% t downloads.hardware3 %} <a href="https://github.com/LedgerHQ/blue-app-monero" target="_blank" rel="noreferrer, noopener">{% t downloads.hardware4 %}</a></p>
+                            </div>
+                        </div>
+                    {% elsif data_downloads.id == "mobilelight" %}
+                        <div class="row">
+                            <div class="col-md-8 col-md-offset-2 col-sm-12 col-xs-12">
+                                <p>{% t downloads.mobilelight1 %} <a href="/community/hangouts/">{% t downloads.mobilelight2 %}</a>{% t downloads.mobilelight3 %}</p>
+                            </div>
+                        </div>
+                        <div class="row center-xs">
+                            <div class="col-xs-6 col-sm-4">
+                                <a href="https://cakewallet.io"><img style="height: 115px;"  src="/img/cakewallet.png" alt="Cake Wallet Logo"></a>
+                            </div>
+                            <div class="col-xs-6 col-sm-4">
+                                <a href="https://monerujo.io"><img style="height: 115px;" src="/img/Monerujo-wallet.png" alt="Monerujo Logo"></a>
+                            </div>
+                            <div class="col-xs-12 col-sm-4">
+                                <a href="https://mymonero.com"><img src="/img/mymonero.png" alt="MyMonero Logo"></a>
+                            </div>
+                        </div>
+                    {% elsif data_downloads.gui_hash == nil and data_downloads.cli_hash != nil %}
+                        <div class="row">
+                            <div class="col-md-8 col-md-offset-2 col-sm-12 col-xs-12">
+                                <h4 id="{{ data_downloads.platform | slugify }}">
+                                     <a href="//downloads.getmonero.org/cli/{{ data_downloads.cli_url }}"> {{ data_downloads.platform }} {% t downloads.clionly %}</a>
+                                </h4>
+                            </div>
+                         </div>
+                        <div class="row">
+                            <div class="col-md-8 col-md-offset-2 col-sm-12 col-xs-12">
+                                <p><strong>SHA256 Hash:</strong></p> 
```

Comment:
This could be localized too. a french version would be `Hachage SHA256`, so a replacement of this `SHA256 Hash:` (and all subsequent occurence) with `{% t downloads.hash %}` would be great.